### ISSUE TITLE
Problemer med duplikate id-er på ikoner. Duplikatene blir mellom pilene i brødsmulestien og ikoner i siden

### ISSUE THREAD

**Issue by PerOlavM** [navikt/nav-dekoratoren_legacy#1410]
Finn ut om vi kan løse dette, eller om det må meldes til designsystemet

**Comment by PerOlavM** [comment]
Case:
https://www.nav.no/person/kontakt-oss/nb/tilbakemeldinger/feil-og-mangler

**Comment by PerOlavM** [comment]
Lagt inn spørsmål på aksel-deisgnsystemet:
https://nav-it.slack.com/archives/C7NE7A8UF/p1679915330247919
